README.md

```diff
@@ -343,6 +343,92 @@ $ plextraktsync --server=Example1 info
 INFO     Enabled 2 libraries in Plex Server: ['Movies', 'TV Shows']
 ```
 
+### Per server configuration
+
+If you want to specify your config per server you can do so inside of `servers.yml`. Within the `config` part of the server configuration you can specify how that specific server should work.
+
+```yml
+  Example1:
+    token: ~
+    urls:
+      - http://localhost:32400
+    config:
+      sync:
+        plex_to_trakt:
+          collection: true
+        trakt_to_plex:
+          liked_lists: false
+```
+Using `sync` in a server config overrides the global sync-config in `config.yml`.
+
+This can also be used to have different configs between different libraries. To be able to do this you specifiy the number of servers you need (most likely equal to the number of different config setups you need). For example:
```

Comment:
```suggestion
This can also be used to have different configs between different libraries. To be able to do this you specify the number of servers you need (most likely equal to the number of different config setups you need). For example:
```

Reply:
LET ME TELL YOu. I misspelled specify so many times last night... I guess you caught one of the ones I missed.